docs/quick-start.md

```diff
@@ -289,13 +289,13 @@ $ find .
 # Package a jar containing your application
 $ sbt package
 ...
-[info] Packaging {..}/{..}/target/scala-2.10/simple-project_2.10-1.0.jar
+[info] Packaging {..}/{..}/target/scala-2.11/simple-project_2.11-1.0.jar
```

Comment:
Actually, I think these references should use `scala-{{site.SCALA_BINARY_VERSION}}` in order to be consistently updated across the docs. These are the only instances I see.